samples/detect.v2beta1.js

```diff
@@ -0,0 +1,822 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const projectId = process.env.GCLOUD_PROJECT;
+const sessionId = require('uuid/v1')();
+const common = require('@google-cloud/common');
+const structjson = require('./structjson.js');
+
+function createKnowledgeBase(projectId, displayName) {
+  // [START dialogflow_create_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const displayName = `your knowledge base display name, e.g. myKnowledgeBase`;
+
+  var formattedParent = client.projectPath(projectId);
+  var knowledgeBase = {
+    displayName: displayName,
+  };
+  var request = {
+    parent: formattedParent,
+    knowledgeBase: knowledgeBase,
+  };
+
+  client
+    .createKnowledgeBase(request)
+    .then(responses => {
+      const result = responses[0];
+      console.log(`Name: ${result.name}`);
+      console.log(`displayName: ${result.displayName}`);
+      return result.displayName;
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END dialogflow_create_knowledge_base]
+}
+
+function getKnowledgeBase(projectId, knowledgeBaseFullName) {
+  // [START dialogflow_get_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
```

Comment:
This is not something we normally expect users to pass in as a paramter

Is there a path helper for getting this? e.g. `client.knowledgeBasePath(projectId, knowledgeBaseName)`?

Normally we accept the individual parts of the path as parameters and construct the path using path helpers (like you did in some of the above samples)